src/interactive.js

```diff
@@ -1,11 +1,27 @@
 import { save, tasks } from './localStorage.js';
 import { render } from './renderTask.js';
 
-export default function checkCompleted(e) {
+function checkCompleted(e) {
   if (e.target.tagName.toLowerCase() === 'input') {
     const selectedTask = tasks.find((task) => task.id === e.target.id);
     selectedTask.complete = e.target.checked;
-    save();
-    render();
+    save(tasks);
+    render(tasks);
   }
 }
+
+function check() {
+  const checkbox = document.querySelectorAll('input[type=checkbox]');
```

Comment:
- [ ] Here as well. I would recommend using `es6` syntax as it is a project requirement that helps you in writing more clean code using ES6 syntax.